velox/exec/fuzzer/WindowFuzzer.cpp

```diff
@@ -53,8 +54,56 @@ void WindowFuzzer::addWindowFunctionSignatures(
 }
 
 const std::string WindowFuzzer::generateFrameClause() {
-  // TODO: allow randomly generating the frame clauses.
-  return "RANGE BETWEEN UNBOUNDED PRECEDING AND CURRENT ROW";
+  auto frameType = [](int value) -> const std::string {
+    switch (value) {
+      case 0:
+        return "ROWS";
+      case 1:
+        return "RANGE";
+      default:
+        VELOX_UNREACHABLE("Unknown value for frame bound generation");
+    }
+  };
+  auto frameTypeValue =
+      boost::random::uniform_int_distribution<uint32_t>(0, 1)(rng_);
+  auto frameTypeString = frameType(frameTypeValue);
+
+  auto frameBound = [&](int value, bool start) -> const std::string {
+    // Generating only constant bounded k PRECEDING/FOLLOWING frames for now.
+    auto kValue =
```

Comment:
We should expand the values here. We could generate -ve, 0 and even very large frame values > 32 bit range for testing more boundary conditions.